redo/api-schema/src/schema/return-read.schema.yaml

```diff
@@ -52,6 +52,25 @@ properties:
     required: [items, provision]
     title: Exchange
     type: object
+  giftCards:
+    description: Gift card(s) created for return
+    items:
+      properties:
+        code:
+          description: Gift card 16 digit code.
+          title: Code
+          type: string
+        externalId:
+          description: Gift card ID.
+          title: ID
+          type: string
+        value:
```

Comment:
To follow convention for our API,

```yaml
amount:
  $ref: ./money.schema.yaml
  title: Amount
```

(And then change the redo implementation to use a `{amount: 123, currency: "USD"}`